src/utilities/auth.ts

```
@@ -0,0 +1,60 @@
+import jwt from 'jsonwebtoken';
+
+// Retrieve the secret key from the environment variable
+const SECRET_KEY = process.env.SECRET_KEY;
+
+if (!SECRET_KEY) {
+  throw new Error('SECRET_KEY environment variable is not set.');
+}
+
+import { IncomingMessage } from 'http';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)